src/auto-start.ts

```diff
@@ -4,23 +4,27 @@ import autoloader from './plugins/autoloader/prism-autoloader';
 Prism.components.add(autoloader);
 
 export const PrismConfig = {
-	// TODO: Update docs
 	/**
-	 * By default, Prism will attempt to highlight all code elements (by calling {@link Prism#highlightAll}) on the
-	 * current page after the page finished loading. This might be a problem if e.g. you wanted to asynchronously load
+	 * By default, Prism will attempt to highlight all code elements (by calling {@link Prism.highlightAll}) on the
+	 * current page after the page finishes loading. This might be a problem if e.g. you wanted to asynchronously load
 	 * additional languages or plugins yourself.
 	 *
 	 * By setting this value to `true`, Prism will not automatically highlight all code elements on the page.
 	 *
-	 * You obviously have to change this value before the automatic highlighting started. To do this, you can add an
-	 * empty Prism object into the global scope before loading the Prism script like this:
+	 * You have to change this value before the automatic highlighting starts. To do this, you can add an
+	 * empty Prism object into the global scope before the Prism script loads like this:
 	 *
 	 * ```js
 	 * window.Prism = window.Prism || {};
 	 * Prism.manual = true;
 	 * // add a new <script> to load Prism's script
 	 * ```
 	 *
+	 * or you can set `data-manual` on the script tag used to load PrismJs:
+	 * ```html
+	 * <script src="prism.js" data-manual></script>
+	 * ```
+	 *
 	 * @default false
 	 * @public
```

Comment:
I'd remove it. If I'm not mistaken, the property visibility is already described in the code (if not specified, it's public by default). TypeDoc also [mentions](https://typedoc.org/documents/Tags._public.html) that this tag is redundant (and not desirable). And the result TypeDoc produces is not that good:
<img width="500" alt="image" src="https://github.com/user-attachments/assets/38757350-4b6b-460c-ac31-87af4df29d5f" />